changelog/23277.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: Add modal to warn users about the behavior when logging in with a root token.
```

Comment:
This is out of date with the new behavior

Reply:
Thanks for catching that!